.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "m2e-core-tests"]
 	path = m2e-core-tests
-	url = https://github.com/tesla/m2e-core-tests.git
-	branch = master
+	url = https://github.com/HannesWell/m2e-core-tests.git
```

Comment:
This has to be reverted before merg, but requires
https://github.com/tesla/m2e-core-tests/pull/186

Reply:
Done.